CMakeLists.txt

```diff
@@ -35,7 +35,8 @@ set(UCPLANNER_SOURCES src/ControlledUnicycle.cpp
                       src/UnicycleFoot.cpp
                       src/FootPrint.cpp
                       src/FootPrintsInterpolator.cpp
-                      src/UnicycleTrajectoryGenerator.cpp)
+                      src/UnicycleTrajectoryGenerator.cpp
+		      src/DcmTrajectoryGenerator.cpp)
```

Comment:
Never mix tabs and spaces for indentation. Mixing spaces and tabs is problematic because the actual width of the tabs depends on the editor, and so if you mix them may result in misaligned in certain editors. See https://github.com/ros-industrial/robotiq/blob/85f1cf2d5cc60d1a7a78384845a9e44283af04f5/robotiq_force_torque_sensor/nodes/rq_sensor.cpp#L246 for an example of misalignment induced by mixing tabs and spaces  out in the wild.

Reply:
Just to clarify, you can typically set your editor to emit a given number of spaces when you hit the tab key. See https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md#c for some rules that we use in YARP and we also try to use in all robotology projects.